packages/connect-analytics/src/types/events.ts

```diff
@@ -28,8 +28,8 @@ export type ConnectAnalyticsEvent =
               referrerEmail?: string;
               method?: string;
               payload?: string[];
-              transportType?: string;
-              transportVersion?: string;
+              transportTypes?: string[];
+              bridgeVersion?: string;
```

Comment:
there are breaking changes to analytics now, am I right? if yes, maybe need to log it somewhere 🤔